packages/management/addons-manager-v2/bundle/config/upstream/webhook-manifests.lib.yaml

```diff
@@ -18,7 +18,7 @@ apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
   labels:
-    tkg.tanzu.vmware.com/addon-webhooks: ""
+    tkg.tanzu.vmware.com/addon-webhooks: "addon-webhooks"
```

Comment:
I don't think you need a value, unless we use the value to different different label values with the same key

Reply:
If you noticed in main.go, we are calling
```
webhooks.InstallNewCertificates(ctx, mgr.GetConfig(), certPath, keyPath, constants.WebhookScrtName, flags.addonNamespace, constants.WebhookServiceName, constants.AddonWebhookLabelKey+"="+constants.AddonWebhookLabelValue)
```
and `constants.AddonWebhookLabelValue` has been changed to `addon-webhooks` in yesterday's Adolfo's cluster deletion PR.

Reply:
Yeah, I didn't realize that. This is fine